docs/conceptual/gpu-arch/mi200-performance-counters.md

```diff
@@ -1,455 +1,591 @@
 # MI200 performance counters and metrics
 <!-- markdownlint-disable no-duplicate-header -->
 
-This document lists and describes the hardware performance counters and the derived metrics available on the AMD Instinct™ MI200 GPU. All hardware performance monitors, and the derived performance metrics are accessible via AMD ROCm™ Profiler tool.
+This document lists and describes the hardware performance counters and the derived metrics
+available on the AMD Instinct™ MI200 GPU. All hardware performance monitors, and the derived
+performance metrics are accessible via AMD ROCm™ Profiler tool.
 
 ## MI200 performance counters list
 
 ```{note}
-Preliminary validation of all MI200 performance counters is in progress. Those with “[*]” appended to the names require further evaluation.
+Preliminary validation of all MI200 performance counters is in progress. Those with “[*]” appended to
+the names require further evaluation.
 ```
 
 ### GRBM
```

Comment:
@SwRaw this is an empty heading; We should add content here or remove the following heading (GRBM counters).